src/components/Book.js

```diff
@@ -13,14 +14,40 @@ function Book({
   };
 
   return (
-    <div className="bookDetails">
-      <span className="bookCategory">{category}</span>
-      <span className="bookTitle">{title}</span>
-      <span className="bookAuthor">{author}</span>
-      <button className="removeBook" type="button" onClick={handleRemoveClick}>
-        Remove
-      </button>
+    <div className={styles.contain}>
+      <div className={styles.bookDetails}>
+        <span className={styles.bookCategory}>{category}</span>
+        <span className={styles.bookTitle}>{title}</span>
+        <span className={styles.bookAuthor}>{author}</span>
+        <div className={styles.containButtons}>
+          <button className={styles.smallButtons} type="button">Comments</button>
+          <button className={`${styles.vertical} ${styles.smallButtons}`} type="button" onClick={handleRemoveClick}>
+            Remove
+          </button>
+          <button className={`${styles.vertical} ${styles.smallButtons}`} type="button">Edit</button>
+        </div>
+      </div>
+      <div className={styles.otherSide}>
+        <div className={styles.progress}>
+          <div className={styles.progressCircle} />
+          <div className={styles.progressOverlay} />
+          <div className={styles.progressValue} />
+        </div>
+        <p className={styles.percentage}>
+          64%
+          <br />
+          <span className={styles.complete}>completed</span>
+        </p>
+        <div className={styles.chapter}>
+          <p className={styles.current}>CURRENT CHAPTER</p>
+          <p className={styles.myChapter}>chapter 17</p>
+          <button type="button" className={styles.update}>UPDATE PROGRESS</button>
```

Comment:
- [ ] The update progress button on your design looks quite different from the one on the given design. Please look at the comparison below and then re style your component to match the given design 

Your design
![image](https://user-images.githubusercontent.com/90222110/226118953-dbd60760-fb50-429d-9298-c42b18862d0b.png)
Given design
![image](https://user-images.githubusercontent.com/90222110/226118965-b0c41f39-c8f3-482b-83fe-2db08fa5cd27.png)

As you can see, you need to use the right blue color, and font color, remove the black borders, and so on. Please work on it 😉